src/components/indexComponents/CategoryDropdown.js

```diff
@@ -0,0 +1,162 @@
+import PropTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+import { css } from '@emotion/react';
+import { Button, Icon, Spinner } from '@newrelic/gatsby-theme-newrelic';
+import Overlay from '@components/Overlay';
+
+import getDisplayName from '@utils/getDisplayName';
+import { QUICKSTARTS_COLLAPSE_BREAKPOINT } from '@data/constants';
+
+const CategoryDropdown = ({ category, categoriesWithCount, handleParam }) => {
+  const [isCategoriesOverlayOpen, setIsCategoriesOverlayOpen] = useState(false);
+  const [loadComplete, setLoadComplete] = useState(false);
+
+  useEffect(() => {
+    setLoadComplete(true);
+  }, [categoriesWithCount]);
```

Comment:
I don't think we need this to be a dependency for this `useEffect`. Since we really just want it to trigger once when the page loads.
```suggestion
  }, []);
```

Reply:
ahh good point. removed them